db/migrate/20211206193923_create_state_routing_target.rb

```diff
@@ -0,0 +1,9 @@
+class CreateStateRoutingTarget < ActiveRecord::Migration[6.1]
+  def change
+    create_table :state_routing_targets do |t|
+      t.string :state_abbreviation, null: false
+      t.references :target,null: false, polymorphic: true, index: true
```

Comment:
nitty, but missing a space after the :target on line 5 😂

Reply:
does polymorphic:true automatically create both the target_id and target_type attrs? thats cool. i dont think i knew that